.eslintrc.json

```diff
@@ -53,4 +56,6 @@
         }
       ]
     }
-  }
\ No newline at end of file
+  }
+
+  
```

Comment:
There are two new lines at the end of this file.  Remove one and ensure there are no trailing spaces after the closing curly brace.